lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -63,7 +63,8 @@ def check_fonts_installed!(config:)
         # Parse the first font in each `font-family` attribute found in all found CSS files.
         # Only the first in each `font-family` font list matters, as others are fallbacks we don't want to use anyway.
         font_families = all_stylesheets.flat_map do |s|
-          File.readlines(s).flat_map do |line|
+          stylesheet_path = resolve_path(s) if can_resolve_path(s)
+          File.readlines(stylesheet_path).flat_map do |line|
```

Comment:
If `can_resolve_path(s)` returns `false`, that means that `stylesheet_path` will be `nil`, and `File.readlines(nil)` will crash with an unclear error.

The `resolve_path` method already handles crashing with a properly-handled `UI.crash!` that prints the path it failed to resolve if that's the case, which would make a better error than having `File.readlines(nil)` crash with an error not mentioning the path that caused the issue. So I think we can just remove the `if can_resolve_path(s)` here to handle those error cases gracefully already?

```suggestion
          stylesheet_path = resolve_path(s)
          File.readlines(stylesheet_path).flat_map do |line|
```

Reply:
Good suggestion. I could see "[!] Unable to locate assets/style.css" error message after applying your suggestion. Thank you!